lua/astrocommunity/pack/laravel/init.lua

```diff
@@ -0,0 +1,89 @@
+      "tpope/vim-dotenv",
+      "MunifTanjim/nui.nvim",
+      "nvimtools/none-ls.nvim",
+      {
+        "AstroNvim/astrocore",
+        opts = function(_, opts)
+          local maps = opts.mappings
+          local prefix = "<Leader>L"
+          maps.n[prefix] = { desc = require("astroui").get_icon("Laravel", 1, true) .. "Laravel" }
+
+          maps.n[prefix .. "a"] = { ":Laravel artisan<CR>", desc = "Artisan" }
+          maps.n[prefix .. "r"] = { ":Laravel routes<CR>", desc = "Routes" }
+          maps.n[prefix .. "c"] = { ":Composer<CR>", desc = "Composer" }
+          maps.n[prefix .. "n"] = { ":Npm<CR>", desc = "Npm" }
+          maps.n[prefix .. "y"] = { ":Yarn<CR>", desc = "Yarn" }
+        end,
+      },
+      { "AstroNvim/astroui", opts = { icons = { Laravel = "󰫐" } } },
+    },
+    event = { "VeryLazy" },
+    config = true,
+  },
+  {
+    "Bleksak/laravel-ide-helper.nvim",
+    dependencies = {
+      {
+        "AstroNvim/astrocore",
+        opts = function(_, opts)
+          local maps = opts.mappings
+          local prefix = "<Leader>Li"
+          maps.n[prefix] = { desc = require("astroui").get_icon("IdeHelper", 1, true) .. "Laravel Ide Helper" }
+
+          maps.n[prefix .. "m"] = {
+            function() require("laravel-ide-helper").generate_models(vim.fn.expand "%") end,
+            desc = "Generate Model Info for current model",
+          }
+          maps.n[prefix .. "M"] = {
+            function() require("laravel-ide-helper").generate_models() end,
+            desc = "Generate Model Info for all models",
+          }
+        end,
+      },
+      { "AstroNvim/astroui", opts = { icons = { IdeHelper = "󱚌" } } },
+    },
+  },
+  {
+    "ricardoramirezr/blade-nav.nvim",
+    dependencies = {
+      "hrsh7th/nvim-cmp",
+    },
+    ft = { "blade", "php" },
+  },
+  {
+    "vim-test/vim-test",
+    cmd = { "TestNearest", "TestFile", "TestLast", "TestClass", "TestSuite", "TestVist" },
+    dependencies = {
+      {
+        "AstroNvim/astrocore",
+        opts = function(_, opts)
+          local maps = opts.mappings
+          local prefix = "<Leader>Lt"
+          maps.n[prefix] = { desc = require("astroui").get_icon("Testing", 1, true) .. "Testing" }
+
+          maps.n[prefix .. "n"] = { ":TestNearest<CR>", desc = "Test Nearest" }
+          maps.n[prefix .. "f"] = { ":TestFile<CR>", desc = "Test File" }
+          maps.n[prefix .. "l"] = { ":TestLast<CR>", desc = "Test Last" }
+          maps.n[prefix .. "c"] = { ":TestClass<CR>", desc = "Test Class" }
+          maps.n[prefix .. "s"] = { ":TestSuite<CR>", desc = "Test Suite" }
+          maps.n[prefix .. "v"] = { ":TestVisit<CR>", desc = "Test Visit" }
+
+          -- Set the strategy to open results in a vertical split
+          vim.g["test#strategy"] = "neovim"
+          vim.g["test#neovim#term_position"] = "vert"
+        end,
+      },
+      { "AstroNvim/astroui", opts = { icons = { Testing = "󰙨" } } },
+    },
+    event = { "VeryLazy" },
+  },
```

Comment:
This should be moved to it's own individual plugin so that users can opt into it. Mainly because it's not laravel/blade specific. Also users might be using neotest for testing along with the neotest-vim-test adapter

Reply:
How do you mean individual plugin, like add their own?

I'll remove it tomorrow :)

I believe laravel.nvim are looking to add some sort of integration with vim-test but im not sure to what extent.

Reply:
Like put that spec into it's own place in AstroCommunity `astrocommunity.test.vim-test` :D Another great addition!

Reply:
then inside of this pack, if it requires further configuration to make it work with Blade/Laravel, you can add an optional spec into the pack that further configures `vim-test` if the user has added it

Reply:
If there is confusion here, I am also glad to help out! If you send how it should be configured and stuff I can help work it into the pack with you

Reply:
I think I've understood. I'll spin up something tomorrow :D

Reply:
This is the PR, This doesn't need any extra config to work phpunit/pest, it just works out of the box with it.

I'll remove it from this pack and the user can add the test plugin if they want too :)

https://github.com/AstroNvim/astrocommunity/pull/1112